frontend/packages/console-dynamic-plugin-sdk/src/extensions/import-environments.ts

```diff
@@ -0,0 +1,28 @@
+import { Extension, ExtensionDeclaration } from '../types';
+
+type ImageEnvironment = {
+  /** Environment variable key */
+  key: string;
+  /** The input field's label */
+  label: string;
```

Comment:
Can we add an optional description here? This would allow us to describe where the "Run command" is saved.